static/js/utils/photometry.js

```diff
@@ -0,0 +1,89 @@
+/* global Plotly */
+
+const filter_mapper = {
+  ztfg: "green",
+  ztfr: "red",
+  ztfi: "orange",
```

Comment:
as discussed though, this was if these 3 filters are the only one used, and it's an incomplete list. Please add the other filters. You can copy/paste the list returned by the configHandler if it need to be static